src/WCSimPhysicsListFactory.cc

```diff
@@ -55,6 +58,7 @@ WCSimPhysicsListFactory::~WCSimPhysicsListFactory()
 void WCSimPhysicsListFactory::ConstructParticle()
 {
   G4VModularPhysicsList::ConstructParticle();
+  G4OpticalPhoton::OpticalPhotonDefinition();
```

Comment:
should not be necessary to define optical photon here

Reply:
👍